libs/dh/wholesale/domain/src/calculation.ts

```diff
@@ -16,29 +16,32 @@
  * limitations under the License.
  */
 //#endregion
+import dayjs from 'dayjs';
```

Comment:
suggestion: let's use `dayjs` from Watt

```suggestion
import { dayjs } from '@energinet-datahub/watt/date';
```

Reply:
b23ef7570c03e07229b82a70b315886349fe0099